.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     executor: gradle_docker
     steps:
       - setup_build_environment
-      - setup_remote_docker:
+      - setup_remote_docker: &latest_remote_docker
           version: 19.03.12
```

Comment:
Should we remove this value?

Reply:
probably instead add the missing comment

```
      # The remote docker version is independent from what's installed in the gradle_docker image and defaults to Docker 17.
      #   Use the latest value from https://circleci.com/docs/2.0/building-docker-images/#docker-version
```